src/main/java/io/boomerang/client/ExternalUserServiceImpl.java

```diff
@@ -71,11 +74,15 @@ private HttpHeaders buildHeaders(String email) {
     final HttpHeaders headers = new HttpHeaders();
     headers.add("Accept", "application/json");
     
-    if (email != null) {
-      headers.add(AUTHORIZATION_HEADER, TOKEN_PREFIX + apiTokenService.createJWTToken(email));     headers.add(AUTHORIZATION_HEADER, TOKEN_PREFIX + apiTokenService.createJWTToken(email));
-    } else {
-      headers.add(AUTHORIZATION_HEADER, TOKEN_PREFIX + apiTokenService.createJWTToken());
-    }
+/*  2026-06-04: Not required since Core migration to new cluster
+ * 
+ *  if (email != null) {
+ *     headers.add(AUTHORIZATION_HEADER, TOKEN_PREFIX + apiTokenService.createJWTToken(email));     headers.add(AUTHORIZATION_HEADER, TOKEN_PREFIX + apiTokenService.createJWTToken(email));
+ *  } else {
+ *     headers.add(AUTHORIZATION_HEADER, TOKEN_PREFIX + apiTokenService.createJWTToken());
+ *  }
+ *  
+ */
```

Comment:
Not following, this is required to call any of the internal Core services. What does Core migration to new cluster have to do with this?